.aspect/bazelrc/bazel6.bazelrc

```diff
@@ -13,3 +13,15 @@ build --reuse_sandbox_directories
 # Avoid this flag being enabled by remote_download_minimal or remote_download_toplevel
 # See https://meroton.com/blog/bazel-6-errors-build-without-the-bytes/
 build --noexperimental_action_cache_store_output_metadata
+
```

Comment:
Were these caused by bzlmod using a newer version of `aspect_bazel_lib` then what we had declared in WORKSPACE? Now bzlmod takes the latest declared dep instead of the first declared...

Reply:
Yep, you nailed it